content/scalers/huawei-cloudeye.md

```diff
@@ -0,0 +1,120 @@
++++
+fragment = "content"
+weight = 100
+title = "Huawei Cloudeye"
+background = "light"
++++
+
+Scale applications based on a Huawei Cloudeye.
+
+**Availability:** v1.0+ | **Maintainer:** Community
```

Comment:
v1.0 has already shipped so this should be 1.1

```suggestion
**Availability:** v1.1+ | **Maintainer:** Community
```

Reply:
done it.